src/Nordea/Components/FlatLink.elm

```diff
@@ -103,6 +109,47 @@ view attributes children (FlatLink config) =
         children
 
 
+inline : String -> String -> Html msg
+inline url text =
+    view [ href url ]
+        [ Html.text text ]
+        (FlatLink
+            { styles =
+                [ Css.display inlineFlex
+                , Css.fontWeight Css.inherit
+                , Themes.color Themes.PrimaryColor Colors.blueNordea
+                , hover [ Themes.color Themes.PrimaryColorLight Colors.blueDeep ]
+                , focus [ Themes.color Themes.PrimaryColorLight Colors.blueDeep ]
```

Comment:
I think the theme colors here are mixed, or is it intentional? (see https://github.com/SGFinansAS/elm-components#themes)

Reply:
Matches the design sketches in the [issue](https://github.com/SGFinansAS/elm-components/issues/188) 🤔

Reply:
I think it always uses the fallback colors here because no theme variables are defined by default in the component library.

I think this is more correct:
```suggestion
                , Themes.color Themes.PrimaryColorLight Colors.blueNordea
                , hover [ Themes.color Themes.PrimaryColor Colors.blueDeep ]
                , focus [ Themes.color Themes.PrimaryColor Colors.blueDeep ]
```